src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/showcase_card.html.twig

```diff
@@ -0,0 +1,41 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% block cms_pages_showcase_card %}
+  {% if not showcaseCardIsClosed %}
+    <div id="cms-pages-showcase-card" class="showcase-card card">
+      <div class="showcase-card__left shape-one helper-card__cms-pages-background">
+        <img src="{{ asset('themes/new-theme/scss/img/helper-card/content@3x.png') }}" class="img-fluid">
+      </div>
+      <div class="showcase-card__right">
+        <h2>{{ 'Create meaningful content'|trans({}, 'Admin.Design.Help') }}</h2>
```

Comment:
Hi @LouiseBonnard ,

does the translation domain looks good here?

Reply:
It is perfect, thank you.